packages/squiggle-lang/__tests__/Reducer/Reducer_MathJs/Reducer_MathJsParse_test.res

```diff
@@ -18,6 +18,12 @@ module MySkip = {
     Skip.test(desc, () => expectParseToBe(expr, answer))
 }
 
+module MyOnly = {
+  let testParse = (expr, answer) => Only.test(expr, () => expectParseToBe(expr, answer))
+  let testDescriptionParse = (desc, expr, answer) =>
+    Only.test(desc, () => expectParseToBe(expr, answer))
+}
+
 describe("MathJs parse", () => {
   describe("literals operators paranthesis", () => {
```

Comment:
`parenthesis`

Reply:
There is no syntax problem there. I am sending tested code